chirp.cc

```diff
@@ -0,0 +1,100 @@
+#include "sl_client.h"
+
+#include <iostream>
+#include <memory>
+#include <stack>
+#include <string>
+
+#include <gflags/gflags.h>
+#include <glog/logging.h>
+#include <grpcpp/grpcpp.h>
+
+using grpc::Channel;
+using grpc::ClientContext;
+using grpc::Status;
+
+// Define flags for gflags
+DEFINE_string(register, "", "Registers the given username");
+DEFINE_string(user, "", "Logs in as the given username");
+DEFINE_string(chirp, "", "Creates a new chirp with the given text");
+DEFINE_string(reply, "-1", "Indicates that the new chirp is a reply to the given id");
+DEFINE_string(follow, "", "Starts following the given username");
+DEFINE_string(read, "", "Reads the chirp thread starting at the given id");
+DEFINE_bool(monitor, false, "Streams new tweets from those currently followed");
+
+int main(int argc, char *argv[]) {
+  // parse flags from command line
+  gflags::ParseCommandLineFlags(&argc, &argv, true);
+
+  // create connection with service layer
+  SL_Client client(grpc::CreateChannel("localhost:50002", grpc::InsecureChannelCredentials()));
+
+  // register flag provided, register username
+  if(!FLAGS_register.empty()) {
```

Comment:
Minor: typically you should have a space after a keyword like `if`.

Reply:
Fixed all instances if this.